Widgets/WidgetViews.swift

```diff
@@ -33,12 +33,12 @@ struct FavoriteView: View {
 
             if let favorite = favorite {
 
-                RoundedRectangle(cornerRadius: 10)
-                    .fill(favorite.needsColorBackground ? Color.forDomain(favorite.domain) : Color.widgetFavoritesBackground)
-                    .shadow(color: Color.black.opacity(0.08), radius: 8, x: 0, y: 2)
-
                 Link(destination: favorite.url) {
 
+                    RoundedRectangle(cornerRadius: 10)
```

Comment:
This ensures the full button size is recognised, otherwise it was just the image and sometimes they were small.